docs/design/expressions/member_access.md

```diff
@@ -200,27 +237,48 @@ fn PrintPointTwice() {
 
   // ✅ OK, `Print` found in type of `p`, namely `Point`.
   p.Print();
-  // ✅ OK, `Print` found in the type `Printable`.
+  // ✅ OK, `Print` found in the type `Printable`, and
+  // `Printable.Print` found in the type of `p`.
   p.(Printable.Print)();
 }
+```
+
+### Facet binding
+
+If any of the above lookups ever looks for members of a
+[facet binding](/docs/design/generics/terminology.md#facet-binding), it should
+consider members of the facet type, treating the facet binding as an
+[archetype](/docs/design/generics/terminology.md#archetype).
```

Comment:
Or maybe just "searches", eg:
```suggestion
If any of the above lookups would search for members of a
[facet binding](/docs/design/generics/terminology.md#facet-binding), it
searches the facet type instead, treating the facet binding as an
[archetype](/docs/design/generics/terminology.md#archetype).
```

Though I do wonder: should we instead say that a search into a facet binding `T:! C` performs a search into the facet `T as C`? Is that different, except in terms of when we perform an `impl` lookup? Notionally I think we want to reuse the `impl`(s) that we discovered for the facet binding rather than doing a fresh `impl` lookup in this case, even though the outcome should be the same.

Reply:
Done